src/components/Nav/Nav.js

```diff
@@ -15,7 +15,9 @@ const Nav = ({ search, handleSearch, handleDisplayAllMovies }) => {
                 name='searchBar'
                 value={search}
                 onChange={handleSearch}
+                onKeyPress={(e) => { e.key === 'Enter' && e.preventDefault(); }}
```

Comment:
Thanks for adding this addition. This will make the user experience so much more seamless.